mapbox/dr-ui
# Adjust spacing and size of subPages in GroupedGuides

This PR will adjust spacing and size of subPages in GroupedGuides to improve readability for longer titles.

Current | Proposed
---|---
<img width="774" alt="Screen Shot 2021-09-29 at 2 52 56 PM" src="https://user-images.githubusercontent.com/2180540/135495675-7b3d3f62-4542-4b62-becb-7b52214a7c9a.png"> | <img width="774" alt="Screen Shot 2021-09-29 at 2 57 45 PM" src="https://user-images.githubusercontent.com/2180540/135495671-a9b98b91-9c9d-4f8e-8f01-5117a84355c3.png">



## How to test

1. Run `nvm use && npm ci && npm start`
2. Visit /PageLayout test case and observe the `Page layout: group index` test case

## QA checklist




Open the test cases app locally on:


## Before merge
